piccolo_admin/endpoints.py

```diff
@@ -349,6 +349,60 @@ class FormConfigResponseModel(BaseModel):
     description: t.Optional[str] = None
 
 
+@dataclass
+class ChartConfig:
+    """
+    Used to specify charts, which are passed into ``create_admin``.
+
+    :param title:
+        This will be displayed in the UI in the sidebar.
+    :param chart_slug:
+        This determines which chart will be displayed.
+    :param chart_type:
+        Available chart types. There are five types: ``Pie``, ``Line``,
+        ``Column``, ``Bar`` and ``Area``.
+    :param data:
+        The data to be passed to the admin ui. The data format must be
+        a ``list of lists`` (eg. ``[["Male", 7], ["Female", 3]]``).
+
+    Here's a full example:
+
+    .. code-block:: python
+
+        async def director_movie_count():
+            movies = await Movie.select(
+                Movie.director.name.as_alias("director"),
+                Count(Movie.id)
+            ).group_by(
+                Movie.director
+            )
+            # Flatten the response so it's a list of lists
+            # like [['George Lucas', 3], ...]
+            return [[i['director'], i['count']] for i in movies]
+
+        director_chart = ChartConfig(
+            title='Movie count',
+            chart_type="Pie", # or Bar or Line etc.
+            data=director_movie_count,
+
+        create_admin(charts=[director_chart])
+
+    """
+
+    def __init__(self, title: str, chart_type: str, data: t.List[t.Any]):
+        self.title = title
+        self.chart_slug = self.title.replace(" ", "-").lower()
+        self.chart_type = chart_type
+        self.data = data
+
+
+class ChartConfigResponseModel(BaseModel):
+    title: str
+    chart_slug: str
+    chart_type: str
+    data: t.List[t.Any]
```

Comment:
If the data returned a list of tuples ... we could define it more precisely like:

```
data: t.List[t.Tuple[str, t.Union[int, float, decimal.Decimal]]]
```

Presumably each element is a string, followed by some kind of number.

Reply:
Returned data must be `list of lists`. I will change the annotation to `t.List[t.List[t.Any]]`.